src/Features/Core/Diagnostics/EngineV1/DiagnosticIncrementalAnalyzer.cs

```diff
@@ -291,7 +291,7 @@ private bool CheckOption(Workspace workspace, string language, bool documentOpen
         {
             try
             {
-                if (!skipClosedFileChecks && !CheckOption(project.Solution.Workspace, project.Language, documentOpened: project.Documents.Any(d => d.IsOpen())))
+                if (!skipClosedFileChecks && !CheckOption(project.Solution.Workspace, project.Language, documentOpened: false))
```

Comment:
the flag means, show me something cheap, don't do anything expensive (CPU, Memory, disk, GC)